ajax/libs/gzip-js/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "gzip-js",
+  "description": "GZIP in pure JavaScript (works in the browser)",
+  "filename": "gzip.min.js",
+  "keywords": [
+    "gzip",
+    "zip",
+    "pure javascript"
+  ],
+  "author": "T. Jameson Little <t.jameson.little@gmail.com>",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/beatgammit/gzip-js.git"
+  },  
```

Comment:
Note that there are two trailing spaces here, right after `}, `
@maruilian11 @sufuf3